src/Features/ERDDAP/Superlatives/index.tsx

```diff
@@ -144,21 +177,13 @@ interface HighestConditionsProps {
  * @param unitSystem Unit system to display values in
  * @param title Title to display
  */
-const HighestConditions: React.FunctionComponent<HighestConditionsProps> = ({
-  platform,
-  timeSeries,
-  unitSystem,
-  title,
-}) => {
+const HighestConditions: React.FunctionComponent<HighestConditionsProps> = ({ platform, timeSeries, unitSystem }) => {
   const dataConverter = converter(timeSeries.data_type.standard_name)
 
   const url = urlPartReplacer(paths.platforms.platform, ":id", platform.id as string)
 
   return (
     <React.Fragment>
-      <Link href={url}>
-        <h6>{title}</h6>
-      </Link>
```

Comment:
Initially I was worried about removing this link, then I realized we had another link to the platform already with the platform name.

Reply:
Yeah, seemed extraneous so I thought it would be okay to move it and remove the link